crates/ruff_linter/src/rules/flake8_raise/rules/unnecessary_paren_on_raise_exception.rs

```diff
@@ -135,11 +147,7 @@ pub(crate) fn unnecessary_paren_on_raise_exception(checker: &Checker, expr: &Exp
         } else {
             diagnostic.set_fix(Fix::applicable_edit(
                 Edit::range_deletion(arguments.range()),
-                if exception_type.is_some() {
-                    Applicability::Safe
-                } else {
-                    Applicability::Unsafe
-                },
+                applicability,
```

Comment:
Can you tell me why it isn't required to check if the fix would delete comments in the `if` branch? If that's indeed the case, I suggest moving the `applicability` override from line 124 into the else branch here.